Update migration docs (#305)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/migrating/import-devices.md

---
title: Import End Devices in The Things Stack
weight: 4
aliases: ["/getting-started/migrating-from-v2/import-devices", "/getting-started/migrating-from-networks/import-devices"]
---

> To import end devices, you need own an application in {{% tts %}}. If you have not created it previously, see [Adding Applications]({{< ref "integrations/adding-applications" >}}) guide for detailed instructions.

Reviewer: ```suggestion
To import end devices, you need access to an application in {{% tts %}}. If you have not created one previously, see the [Adding Applications]({{< ref "integrations/adding-applications" >}}) guide for detailed instructions.
```
